hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/HttpFSServerWebServer.java

```diff
@@ -178,6 +179,10 @@ public URL getUrl() {
     }
   }
 
+  public HttpServer2 getHttpServer() {
```

Comment:
Was this only added for testing? If so, please add a VisibleForTesting annotation over the method.

Reply:
Done.